packages/stencil/components/pagination/src/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html dir='ltr' lang='en'>
+<head>
+  <meta charset='utf-8'/>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0'/>
+  <title>Dev osds-pagination</title>
+
+  <script type="module">
+    !window.ods && (window.ods = {config: {logging: {active: true}}});
+  </script>
+
+  <script type='module' src='/build/osds-pagination.esm.js'></script>
+  <script nomodule src='/build/osds-pagination.js'></script>
+  <link rel="stylesheet" href="/build/osds-pagination.css">
+  <style>
+    h3 {
+      font-family: 'Gill Sans', Georgia;
+    }
+  </style>
+</head>
+<body>
+
+<h3>3rd page and previous</h3>
+<div style="text-align:center">
+<osds-pagination default-current=1 current=3 total-pages=21 id="pagination1"></osds-pagination>
+<p id="select2-info" style="font-style: italic;">21 pages to display, current page 3</p>
```

Comment:
Please update the id with correct naming, select => pagination
Same for followings

Reply:
Done ✅